images/scripts/rhel.setup

```diff
@@ -266,6 +270,7 @@ if [ "$DO_BUILD" -eq 1 ]; then
         case "$IMAGE" in
             rhel-8-1) REPO="rhel-8.1.0-build" ;;
             rhel-8-2) REPO="rhel-8.2.0-build" ;;
+            centos-8-stream) REPO="rhel-8.2.0-build" ;;
```

Comment:
That looks wrong -- we don't want to use the RHEL mock build on a CentOS image. (And also can't, that's a license violation). This needs to use an EPEL/CentOS build root.

Reply:
ack, changing this. Wasn't sure what to do with it.